andela/ah-django-unchained
# #162948859 Implement tracking of comment history

#### What does this PR do?
Allows users to track history of comments they make on the platform after editing

#### Description of Task to be completed?
Authenticated users should be able to track edits they make to a particular comment.

#### How should this be manually tested?

- Clone the repo locally
- cd ah-django-unchained
- git checkout ft-track-comment-history-162948859
- get started with the app as stated in the README section
- to add an article ensure you obtain a token from logging in or signing up if not signed up
- Using the `POST api/articles/` endpoint, create an article
- Using the ` POST api/articles/<slug>/comments/` endpoint, add a comment to the article where `<slug>` is the slug generated after the article is posted 
- Using the `PUT api/articles/<slug>/comments/<id>/` endpoint,  update the comment you created where `<id>` is the id of the comment
- Using the ` GET api/articles/<slug>/comments/<id>/history/` endpoint, view the history of the comment you created then updated.
#### Screenshots
![image](https://user-images.githubusercontent.com/27801956/52112674-429a7480-2618-11e9-8b3d-1d6f92dc0c89.png)

#### Relevant Pivotal Tracker Stories

[#162948859](https://www.pivotaltracker.com/story/show/162948859)
